<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTilesetsDock</class>
 <widget class="QDockWidget" name="QTilesetsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>118</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tilesets</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tilePicker">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="newButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="S2PEditor.qrc">
          <normaloff>:/Common/assets/images/GUI_Icons/add-1.png</normaloff>:/Common/assets/images/GUI_Icons/add-1.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomInButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="S2PEditor.qrc">
          <normaloff>:/Common/assets/images/GUI_Icons/zoom-in.png</normaloff>:/Common/assets/images/GUI_Icons/zoom-in.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOutButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="S2PEditor.qrc">
          <normaloff>:/Common/assets/images/GUI_Icons/zoom-out.png</normaloff>:/Common/assets/images/GUI_Icons/zoom-out.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="zoomSpin">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="collisionCheckBox">
        <property name="text">
         <string>Collision</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="S2PEditor.qrc"/>
 </resources>
 <connections/>
</ui>
